modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
+
+[NOTE]
+====
+When you list VMIs using the CLI, all VMIs including standalone VMIs and VMIs that are owned by VMs and other objects display. The web console displays standalone VMIs. VMIs that are owned by VMs or other objects are excluded from display in the web console.
+====
+
+
+.Procedure
+
+. Click *Workloads > Virtualization* from the side menu. A list of VMs and standalone VMIs display. You can identify standalone VMIs by the dark colored badges that display next to the virtual machine instance names.
+
+. For details, click on a standalone VMI listed in the  *Name* column. The *Virtual Machine Instance Overview* page displays.
```

Comment:
s/click on/click

The second step is optional, because this procedure is just for listing VMIs. I'm not sure it needs to be included, but if you keep it, mark it as optional.